transaction-status/Cargo.toml

```diff
@@ -13,7 +13,7 @@ edition = { workspace = true }
 Inflector = { workspace = true }
 base64 = { workspace = true }
 bincode = { workspace = true }
-borsh = { workspace = true }
+borsh = { version = "0.10.3" }
```

Comment:
I'm trying to remember fully how this is used...
Will we need to bump it at some point? Or no, because ATA is deployed with v0.10?

Reply:
Sorry, I should have put a comment here. This is to deserialize the ATA instructions, so we need to match the version of borsh that it uses. Since the current crate is on 0.10, we have to keep using that.

When we update ATA to use borsh 1.2 and start using that crate in the monorepo, we can go back to the workspace version.

I added a comment in 52e2325e449f14c1405e4001bd412e037c733eba